codersdk/parameters.go

```diff
@@ -78,6 +78,13 @@ type ParameterSchema struct {
 
 // CreateParameterRequest is used to create a new parameter value for a scope.
 type CreateParameterRequest struct {
+	// CopyFromParameter allows copying the value of another parameter.
+	// The other param must be related to the same template_id for this to
+	// succeed.
+	// No other fields are required if using this, as all fields will be copied
+	// from the other parameter.
+	CopyFromParameter uuid.UUID `json:"copy_from_parameter,omitempty" validate:"uuid4"`
```

Comment:
What do you think about `CopyFrom` instead? Or `CloneID`? No specific reason, just preference.

Reply:
`CloneID` I like.